modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/paths/APIBuilder.path

```diff
@@ -100,6 +105,11 @@
 	<td>//input[(@placeholder='Enter name.')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ENTER_PATH</td>
```

Comment:
ENTER_NAME and ENTER_PATH could have had the same path with key_value in placeholder